docs/basics/sso/migration-from-deprecated-sso.md

```diff
@@ -0,0 +1,75 @@
+| SP-initiated Flow                 | The new SSO supports both [Service Provider (SP) initiated and Identity Provider (IdP) initiated SSO](/basics/sso/logging-in-via-sso), whereas the legacy SSO only supports IdP-initiated flow.                                                                                                                                                                                                                                                                                                                                                                                            |
+| Email Domains                     | Assign your [company email domains](/basics/sso/setting-up-sso/#email-domains) to your SSO integration at Sauce Labs to allow for provisioning of new accounts via SP-initiated flow. <br/><br/> The legacy SSO does not require email domains as it does not support SP-initiated flow.                                                                                                                                                                                                                                                                                                   |
+| JIT Provisioning                  | In the new SSO, JIT (Just-In-Time) provisioning is enabled by default and cannot be disabled. This means that if a user from your Identity Provider (IdP) does not have a Sauce Labs account, one will be automatically created for them. <br/><br/> However, in the legacy SSO, some organizations requested to disable this feature. In such cases, it's important to note that users from your IdP who do not have accounts in Sauce Labs will still be able to provision accounts automatically. Please keep this in mind during the migration process from legacy SSO to the new SSO. |
+| Enforce SAML SSO                  | This [setting](/basics/sso/setting-up-sso/#enforce-saml-sso) is also present in the new SSO, but it is common to both the new SSO and the legacy SSO. When turned on, it allows authentication only via either the new SSO or the legacy SSO. When turned off, users can also log in via username and password, in addition to SSO authentication.                                                                                                                                                                                                                                         |
```

Comment:
suggest to change legacy to depreciated in all places for consistency (or depreciated to legacy if you prefer) 
```suggestion
| Enforce SAML SSO                  | This [setting](/basics/sso/setting-up-sso/#enforce-saml-sso) is also present in the new SSO, but it is common to both the new SSO and the legacy SSO. When turned on, it allows authentication only via either the new SSO or the legacy SSO. When turned off, users can also log in via username and password, in addition to SSO authentication.                                                                                                                                                                                                                                         |
```

Reply:
ok, for consistency I will not use "legacy" but I thought that the word "deprecated" is more appropriate in this context than "depreciated". What do you suggest?

source: https://www.baeldung.com/cs/deprecated-vs-obsolete

Reply:
yes, that was an autocorrect typo - this should be deprecated.

Reply:
fixed